### PR TITLE
File Browser : add file permissions and hidden files

### PR DESCRIPTION
**Pull request by fviale**
- add file permissions in the form "rwx" to file list
 - add check box to hide/show files or directories starting with .
 - improve error messages in case of unauthorized file or folder operations

### CODE REVIEW

**Review by tao-xinxiu — APPROVED** [review]
LGTM

File: app/scripts/proactive/templates/file-browser-template.html [hunk 1]
```diff
@@ -47,27 +50,36 @@ <h5>
...
+                        <td> <%- directories[index].rights %> </td>
                         <td> </td>
                         <td> Folder </td>
                         <td> <%- directories[index].modified %> </td>
                     </tr>
-                    <% } %>
-                    <% for (var index in files) { %>
+                    <%   }
+                       } %>
+                    <% for (var index in files) {
```
Comment (MaelAudren) [inline app/scripts/proactive/templates/file-browser-template.html:71]: I don't think you need to close then open the `%>` and `<%` tokens here.

**Review by MaelAudren — APPROVED** [review]

File: app/scripts/proactive/templates/file-browser-template.html (hunk 1 above)
Comment (fviale) [reply app/scripts/proactive/templates/file-browser-template.html:71]: yes, right

**Comment by fviale** [comment]
Retest this please

**Pull request merged**